index.js

```diff
@@ -4030,6 +4037,41 @@
   S.splitOn =
   def('splitOn', {}, [$.String, $.String, $.Array($.String)], splitOn);
 
+  //# sortBy :: (a -> a -> Ordering) -> Array a -> Array a
+  //.
+  //. Returns a copy of the array, sorted according to the comparator
+  //. function. The comparator function is expected to be curried,
+  //. and will receive two values at a time. The comparator must return
+  //. a value of type Ordering, which can be any of the following strings:
+  //. - 'LT': when the first value received goes **before** the second value.
+  //. - 'GT': when the first value received goes **after** the second value.
```

Comment:
Rather than describe the behaviour in terms of before/after, I think it's sufficient to use less/greater than. For example, _the comparator should return `'LT'` if the first value is less than the second value_.